### PR TITLE
Added NuGet Video "Install and Use a NuGet Package with the .NET CLI"

### PR DESCRIPTION
**Pull request by chgill-MSFT**
In the preview it doesn't show the video, but it seems to appear fine when published. This is the exact type/style of embedding used in other docs with videos such as:
https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/essentials/includes/xamarin-show-essentials.md

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[a9c3f7d](https://github.com/NuGet/docs.microsoft.com-nuget/commits/a9c3f7d591ab70ecaed5b0097532e1ccbbfc9292)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/chgill-MSFT-patch-2/docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-using-the-dotnet-cli?branch=pr-en-us-1883) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C10%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002102155303651-1883%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=HIxBM7IO2cdLJTb27ZDl63CsLoV%2F%2BhB6mhuzaOJXUoE%3D&st=2020-02-10T21%3A51%3A06Z&se=2020-03-12T21%3A56%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[3a08069](https://github.com/NuGet/docs.microsoft.com-nuget/commits/3a08069695f30c61bbac92ac55eefde3ba730786)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/chgill-MSFT-patch-2/docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-using-the-dotnet-cli?branch=pr-en-us-1883) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C10%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002102200351091-1883%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=npzIYiOdQCNb%2FqohVrqwYGV3%2F0j5PRO%2F4vW8Sev7pR0%3D&st=2020-02-10T21%3A56%3A18Z&se=2020-03-12T22%3A01%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md [hunk 1]
```diff
@@ -110,3 +110,9 @@ To explore more that NuGet has to offer, select the links below.
 - [Overview and workflow of package consumption](../consume-packages/overview-and-workflow.md)
 - [Finding and choosing packages](../consume-packages/finding-and-choosing-packages.md)
 - [Package references in project files](../consume-packages/package-references-in-project-files.md)
+
+## Related video
+
+> [!Video https://channel9.msdn.com/Series/NuGet-101/Install-and-Use-a-NuGet-Package-with-the-NET-CLI-3-of-5]
```
Comment (zivkan) [inline docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md]: You're missing `/player` at the end of the video URL. Additionally, next time please create 1 PR with all the changes. It reduces review workload.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[e9d8604](https://github.com/NuGet/docs.microsoft.com-nuget/commits/e9d8604499c9b15271c57a16b4231418531ba37e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/chgill-MSFT-patch-2/docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-using-the-dotnet-cli?branch=pr-en-us-1883) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002192124567924-1883%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=dN1dCeCy10MSJZx7aGYa3uhb7i8iN0uOPIfcPiIit30%3D&st=2020-02-19T21%3A20%3A39Z&se=2020-03-21T21%3A25%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[957093f](https://github.com/NuGet/docs.microsoft.com-nuget/commits/957093ff5a3baac911cbf61cc41cea99be28484e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/chgill-MSFT-patch-2/docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-using-the-dotnet-cli?branch=pr-en-us-1883) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002192203239396-1883%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=SSjdohCu21Aew%2FJknTCG8yp5EB9imp89lq3rpVkZPgQ%3D&st=2020-02-19T21%3A59%3A03Z&se=2020-03-21T22%3A04%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**